python/tvm/contrib/hexagon/build.py

```diff
@@ -273,7 +273,11 @@ class HexagonLauncherAndroid(HexagonLauncherRPC):
     ]
 
     def __init__(
-        self, serial_number: str, rpc_info: dict, workspace: Union[str, pathlib.Path] = None
+        self,
+        serial_number: str,
+        rpc_info: dict,
+        workspace: Union[str, pathlib.Path] = None,
+        adb_socket: str = "tcp:5037",
```

Comment:
This should be in `rpc_info`.  The simulator would simply not use it, and it would keep the function interfaces unchanged.

Reply:
good point! fixed it.